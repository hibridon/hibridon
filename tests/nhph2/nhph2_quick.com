inp=Nhph2q.inp
job=Jobq
show
run
printc

