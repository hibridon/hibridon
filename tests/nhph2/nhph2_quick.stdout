
 --------------------------------------------------------------------------
           HIBRIDON SCATTERING CODE V 5.00

      AUTHORS: M. ALEXANDER, D. MANOLOPOULOS, H.-J. WERNER, B. FOLLMEG,
               P. DAGDIGIAN
 CONTRIBUTORS: D. LEMOINE, P. VOHRALIK, G. COREY, R. JOHNSON, T. ORLIKOWSKI,
          A. BERNING, A. DEGLI-ESPOSTI, C. RIST, B. POUILLY, J. KLOS, Q. MA,
          G. VAN DER SANDEN, M. YANG, F. DE WEERD, S. GREGURICK, F. LIQUE
 --------------------------------------------------------------------------
           BUILD CONFIGURATION:
CMAKE BUILD DATE: 2021-10-14 at 08:28
CMAKE BUILD TYPE: Debug
CODE GIT REVISION: c5e68b5
SYS: Linux 5.4.0-88-generic x86_64
COMPILER: /usr/bin/gfortran
COMPILE OPTIONS: 
 --------------------------------------------------------------------------
            CURRENT HARDWARE CONFIGURATION:

Model name:                      Intel(R) Core(TM) i5-8350U CPU @ 1.70GHz
CPU MHz:                         3050.784
Socket(s):                       1
Core(s) per socket:              4
L2 cache:                        1 MiB
L3 cache:                        6 MiB
Hyper-Threading Technology: Enabled
MemTotal:       16272252 kB
 --------------------------------------------------------------------------
 Hibridon>  Hibridon>  Hibridon>      *** Parameters (scattering):
 JTOT1  =   0        JTOT2  =   1        JTOTD  =   1        JLPAR  =   0
 NERG   =   1        NUMAX  =   0        NUMIN  =   0        NUD    =   1
 LSCREEN=  80        IPRINT =  -1
 FSTFAC =  1.500     RINCR  =  1.000     RCUT   = -100.0     RENDAI =  60.00    
 RENDLD =  10.00     RSTART =  4.250     SPAC   = 0.1500     TOLAI  =  1.050    
 XMU    =   1.77703
 NOUT: 20; J1/J2-OUT: 00  10  20  30  40  50  60  70  80  90  01  11  21  31  41  51  61  71  81  91  
 INDOUT:    1    2    3
     *** 3SG-1SG system parameters:
 J1MAX  =   4        J2MIN  =   0        J2MAX  =   2        IPOTSY2=   2
 B1ROT  =  16.34     D1ROT  =  0.000     LAMBDA = 0.9199     GAMMA  =-5.4853E-02
 B2ROT  =  59.32    
 J1/J2: 
     *** Flags:
 AIRYFL= T    BASTST= F    BATCH = F    CHLIST= T    CSFLAG= F    FLAGHF= F
 FLAGSU= F    IHOMO = F    IPOS  = F    LOGDFL= T    NOPRIN= T    NUCROS= F
 PHOTOF= F    PRAIRY= F    PRLOGD= F    PRPART= F    PRSMAT= F    PRT2  = F
 PRXSEC= F    READPT= F    RSFLAG= F    T2TEST= F    TWOMOL= T    WAVEFL= F
 WRPART= F    WRSMAT= T    WRXSEC= T    BOUNDC= F
 ** Maximum Channels:  214; Maximum Anisotropic Terms:  2000
 ** Energies:       50.000000
 ** Label:      NH - H2 RCCSD(T)-F12a(T) AVTZ PES by Y.         
 ** Pot name:      NH - H2 PES                                     
 ** Input File:  Nhph2q.inp
 ** Output file: Outpt
 ** Jobname:     Jobq
 Hibridon>  ** J =    0 JLPAR = 1 FINISHED;  CPU: 00:00:00   WALL: 00:00:00   DATE: Oct 14 08:32:52 2021
 ** J =    1 JLPAR = 1 FINISHED;  CPU: 00:00:01   WALL: 00:00:01   DATE: Oct 14 08:32:53 2021
 ** J =    0 JLPAR =-1 FINISHED;  CPU: 00:00:01   WALL: 00:00:01   DATE: Oct 14 08:32:53 2021
 ** J =    1 JLPAR =-1 FINISHED;  CPU: 00:00:01   WALL: 00:00:02   DATE: Oct 14 08:32:54 2021
 ===============================================================================
 **** END OF CALCULATION ****
      MAXIMUM NUMBER OF CHANNELS USED WAS:    93
      TIMING:  ELAPSED 00:00:02 / CPU 00:00:01 / MP RATIO   93.1 %
      CURRENT DATE:  Oct 14 08:32:54 2021
 ===============================================================================
 Hibridon> 
 INTEGRAL CROSS SECTIONS READ FROM FILE Jobq1.ics                               
 WRITTEN:   Oct 14 08:32:54 2021
 LABEL:     NH - H2 RCCSD(T)-F12a(T) AVTZ PES by Y.         
 POT NAME:  NH - H2 PES                                     

 ** INTEGRAL CROSS SECTIONS; IEN= 1 **
    RMU=   1.7770  E=    50.00  JLPAR=-1  JTOT-1=  0  JTOT-2=   1  JTOT-D=  1
 ** CC CALCULATION, JLPAR=-1 **

 ** COLUMN HEADINGS ARE INITIAL STATES, ROW HEADINGS ARE FINAL STATES **

            J=    0        10        20        10
   J    I | I=    1         1         1         2
 
   10    1   1.942E-03  4.501E+00  5.539E-03  3.596E-02
    0    1   6.394E+00  2.386E-04  7.551E-02  4.090E-05
   20    1   3.584E-01  3.229E-03  7.419E+00  3.462E-01
   10    2   1.108E-04  1.197E-02  1.976E-01  1.277E+01
 Hibridon>  Hibridon> 
